spring-cloud-azure-samples/spring-cloud-azure-servicebus-sample/src/main/java/example/WebController.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package example;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author Warren Zhu
+ */
+@RestController
+public class WebController {
+
+    private final ServiceBusQueueApplication.ServiceBusOutboundGateway messagingGateway;
```

Comment:
when is messagingGateway injected? do we need this: messagingGateway = appCtx.getBean(ServiceBusOutboundGateway.class)?

Reply:
gateway is injected by `@EnableIntegration`